Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -435,6 +443,10 @@ export default class ProfileView extends Vue {
                 this.isLoading = false;
             });
     }
+
+    private get PHN(): string {
```

Comment:
This should be lowercase: `phn()` rather than `PHN()`.

Reply:
I hope that one day the margins will be consistent.